libs/java.json

```diff
@@ -14,6 +14,7 @@
   {"id":"java.pull-to-refresh","imports":["com.handmark.pulltorefresh."],"name":"Pull To Refresh","repo":"chrisbanes/Android-PullToRefresh","tags":["pull-to-refresh"],"tech":["android-ui"],"status":"model-ready"},
   {"id":"java.rx-binding","imports":["com.jakewharton.rxbinding2."],"name":"RxBinding","repo":"JakeWharton/RxBinding","tags":["reactive"],"tech":["android-ui"],"status":"model-ready"},
   {"id":"java.sliding-menu","imports":["com.jeremyfeinstein.slidingmenu."],"name":"SlidingMenu","repo":"jfeinstein10/SlidingMenu","tags":["menus"],"tech":["android-ui"],"status":"model-ready"},
+  {"id":"java.facebook-android-sdk","imports":["com.facebook"],"name":"Facebook SDK for Android","repo":"facebook/facebook-android-sdk","tags":["facebook"],"tech":["api"],"status":"awaiting-model"},
```

Comment:
can we call "Facebook Android SDK" (lose 'for') because we don't have much real estate when we show it.